js/main.js

```diff
@@ -0,0 +1,51 @@
+
+}
+$('#form').addEventListener('submit', e => {
+    e.preventDefault();
+    if ($('#username').value == '') {
+        $('.error-username').innerHTML = 'Trống!!!'
+    }
+    else if (!checkName($('#username').value)) {
+        $('.error-username').innerHTML = 'Không có kí tự đặc biệt,số'
+    }
+    else $('.error-username').innerHTML == '';
+
+    if ($('#email').value =='') {
+        $('.error-email').innerHTML = 'Trống!!!'
+    }
+    else if (!checkEmail($('#email').value)) {
+        $('.error-email').innerHTML = 'Email sai định dạng!!!'
+    }
+    else $('.error-email').innerHTML == '';
```

Comment:
```suggestion
    else $('.error-email').innerHTML === '';
```
nên tạo 1 hàm required thay vì check thủ công như này nhé